plugins/modules/iptables_state.py

```diff
@@ -233,6 +235,36 @@
       ]
     }
   returned: always
+tables_after:
+  description:
+    - The iptables on the system after the module has run, separated by table.
```

Comment:
This is a new feature. Would you mind moving this to a separate PR so the bugfix can be backported? (Or alternatively move the bugfix to a separate PR.)